pom.xml

```diff
@@ -79,13 +79,13 @@
 		<!-- dependency versions -->
 		<junit.version>4.13.2</junit.version>
 		<storm-client.version>2.6.0</storm-client.version>
-		<jackson.version>2.15.2</jackson.version>
+		<jackson.version>2.16.0</jackson.version>
 		<tika.version>2.9.1</tika.version>
-		<mockito.version>5.7.0</mockito.version>
-		<jetbrains.annotations.version>24.0.1</jetbrains.annotations.version>
-		<log4j2.version>2.20.0</log4j2.version>
+		<mockito.version>5.8.0</mockito.version>
+		<jetbrains.annotations.version>24.1.0</jetbrains.annotations.version>
+		<log4j2.version>2.22.0</log4j2.version>
```

Comment:
Storm is on 2.21.1 
https://github.com/apache/storm/blob/ae3a96e762095553311d9e335f7505c0b351d810/pom.xml#L109
stay on that